### ISSUE TITLE
Support for prerequisite features

### ISSUE THREAD

**Issue by yrashk** [lburgazzoli/gradle-karaf-plugin#21]
This is something we should probably consider supporting:

```xml
<features xmlns="http://karaf.apache.org/xmlns/features/v1.4.0" name="app-2.0.0">
	<feature name="external-libs" version="2.0.0" description="External libs">
		<details>External dependencies</details>
		<feature prerequisite="true">wrap</feature>
		<bundle start-level="80">wrap:mvn:net.sf.ehcache/ehcache-core/2.6.11$Bundle-SymbolicName=ehcache-core&amp;Bundle-Version=2.6.11</bundle>
	</feature>
</features>
```

It got documented in 4.0.5 https://github.com/apache/karaf/blob/6a9ca438ccf284cd5a36b2e51768172d21290110/manual/src/main/asciidoc/developer-guide/creating-bundles.adoc

https://github.com/apache/karaf/commit/6a9ca43

Thoughts?

**Comment by lburgazzoli** [comment]
I will add the possibility to set attributes to features as for bundle, stay tuned ;)

**Comment by lburgazzoli** [comment]
Available as v0.0.28 

```groovy
karaf {
    features {
        feature {
                feature 'dependencyFeatureName1'
                feature('dependencyFeatureName2') {
                    version = "5.6.7"
                    dependency = true
                }
                feature('dependencyFeatureName3') {
                    prerequisite = true
                 }
        }
    }
}
```
